test-schema.json

```diff
@@ -56,6 +70,18 @@
                 }
             },
             "additionalProperties": false
+        },
+        "spec": {
+            "properties": {
+                "section": {
+                    "type": "string",
+                    "pattern": "^\\d+(\\.\\d+)*\\.$"
+                },
+                "para": {
```

Comment:
I don't like abbreviations personally, but also I don't think paragraph numbers are that much fun, they're hard both for the person writing the thing *as well* as the reader, both of whom have to count to find what they want.

Ideally everything should have its own anchor and/or section link though certainly they do not today, but I think probably the right thing to do there is `quote` -- i.e. a field that simply allows someone to copy-paste some substring from the spec to put here, and then they copy paste a sentence or so.

Reply:
Maybe this should be used sparingly, like only when the section alone isn't sufficient.

Reply:
Yes definitely agree there!

Reply:
Got this one too